GameOfRunes/GameOfRunes/Utils/RealmStorage.swift

```diff
@@ -0,0 +1,118 @@
+//
+//  Storage.swift
+//  GameOfRunes
+//
+//  Created by Jermy on 31/3/20.
+//  Copyright © 2020 TeamHoWan. All rights reserved.
+//
+
+import RealmSwift
+
+class RealmStorage: Storage {
+    private var realm: Realm!
+    
+    init() {
+        setUpDb()
+    }
+    
+    private func setUpDb() {
+        print("Filepath:", Realm.Configuration.defaultConfiguration.fileURL ?? "Cannot be found")
+        
+        do {
+            realm = try Realm()
+            print("Successfully load Realm database")
+            return
+        } catch {
+            // cleaning
+            print(error.localizedDescription)
+            print("Proceed to reset to clean slate")
+            resetRealm()
+        }
+        
+        // try again
+        do {
+            realm = try Realm()
+            print("Successfully load Realm database")
+        } catch {
+            print(error.localizedDescription)
```

Comment:
should this be `fatalError`?

Reply:
hmm, i think shouldnt crash the game when the DB not working lololol